### DISCUSSION TITLE
Building the vehicle of the future on GitHub with Arm - DEV2770M

### DISCUSSION THREAD

**Discussion by jricciardi** [community/community#74937] [Other Feature Feedback, Questions, & Ideas]
Thursday, Nov 911:15 AM - 11:45 AM PST
The Software Defined Vehicle is creating a once-in-a-lifetime disruption in today’s automotive industry, with each car projected to be driving around with over one billion lines of code working together. Car manufacturers and their suppliers are moving from discrete components that plug and play to rich, complex components with APIs. In this talk, Christopher Rumpf, Senior Director of Arm’s Automotive business, will show you how GitHub’s developer, enterprise security, and AI services are perfectly suited for embedded software teams that leverage the Arm instruction set. Rumpf will give you a tour of how the automotive supply chain is collaborating safely and securely with modern, efficient standardized tools and much more flexible methodologies, which are allowing the automotive industry to innovate faster than ever before.

Christopher Rumpf
Senior Director - Automotive Software & Solutions, Arm

> With over 20 years of leadership at prominent semiconductor companies, Christopher has held several roles, creating a unique understanding of strategy, marketing, r&d, sales, and support. Christopher is currently responsible for Arm’s Automotive software and solution product strategies, focusing on IoT and automotive since 2004.

![Screenshot 2023-11-09 at 11 17 37 AM](https://github.com/community/community/assets/31521880/0dd48e15-f38a-4ba8-943f-1602dee12a10)

**Comment by jricciardi** [comment]
Welcome to the main stage, Christopher Rumpf!

**Comment by jricciardi** [comment]
Not only had this announcement gotten applause heard through the whole convention center earlier today, this is a whole talk about Arm (with demos to come)!
![Screenshot 2023-11-09 at 11 18 59 AM](https://github.com/community/community/assets/31521880/059cd986-f2dd-4331-b809-00be0c79c1aa)

**Comment by jricciardi** [comment]
In the automotive industry, Christopher says "Software integration and test will consume 40% of budgets in 2030" 🤯 

![Screenshot 2023-11-09 at 11 22 21 AM](https://github.com/community/community/assets/31521880/c1aa3866-7032-404e-8bd3-da72f59aa0c2)

**Comment by jricciardi** [comment]
Oh, I should have dropped the link! Learn all about Arm-based hosted runners over on the official GitHub Blog! [Accelerate your CI/CD with Arm-based hosted runners in GitHub Actions](https://github.blog/changelog/2023-10-30-accelerate-your-ci-cd-with-arm-based-hosted-runners-in-github-actions/)

Best part? These runners will be entering private beta in January 2024 - and you can sign up for the private beta waitlist at the bottom of that blog post ❤️

**Comment by jricciardi** [comment]
"In the Cloud, at the Edge"

![Screenshot 2023-11-09 at 11 29 53 AM](https://github.com/community/community/assets/31521880/8f21068d-431a-4a99-8ff0-a377565b31e0)

**Reply by dalesimpson** [reply]
Is anyone else having flashbacks to reading [The Soul of a New Machine](https://www.wikiwand.com/en/The_Soul_of_a_New_Machine) in the framing portion of this talk? 😄

**Reply by jricciardi** [reply]
👀 I'm not familiar with the book, but that's an 80s-as-heck plot description and I'm here for it

**Comment by jricciardi** [comment]
👋  I'm trying to figure out how to snippet this demo, but I'm coming up dry — there's a LOT of cool information here! I'd recommend checking out the recording of this session for the full demo. The recording should go up by end of today!

**Comment by jricciardi** [comment]
✅ Improved price performance 
✅ Lowered carbon impact
✅ Do more with the same budget

![Screenshot 2023-11-09 at 11 42 37 AM](https://github.com/community/community/assets/31521880/1c1cca57-735f-4790-bbd3-9cc55eaea096)

**Comment by jricciardi** [comment]
That's a wrap folks! We'll post the recording for this session once it is uploaded later today 🎉 That's one of the most in-depth, technical talks we've had at Universe 2023 so far, and there's a ton of great stuff in the demo portion.

Here's that killer last slide:

![Screenshot 2023-11-09 at 11 44 42 AM](https://github.com/community/community/assets/31521880/a11eeffe-eed8-4ba0-ae5b-c09635da6a20)
